commands/InjectedCommandGroup.java

```diff
@@ -1,49 +1,38 @@
 package org.usfirst.frc4904.standard.commands;
 
+import edu.wpi.first.wpilibj2.command.CommandGroupBase;
 
-import edu.wpi.first.wpilibj.command.Command;
-import edu.wpi.first.wpilibj.command.CommandGroup;
+public abstract class InjectedCommandGroup extends CommandGroupBase {
+	private final CommandGroupBase previous;
 
-public abstract class InjectedCommandGroup extends CommandGroup {
-	private final Command previous;
-	
-	public InjectedCommandGroup(Command previous) {
-		super();
+	public InjectedCommandGroup(String name, CommandGroupBase previous) {
+		super(); // TODO: is this redundant? https://github.com/RoboticsTeam4904/standard/pull/254#discussion_r366097218
+		setName(name);
 		this.previous = previous;
 	}
-	
-	public InjectedCommandGroup(String name, Command previous) {
-		super(name);
-		this.previous = previous;
+
+	public InjectedCommandGroup(CommandGroupBase previous) {
+		this("InjectedCommandGroup", previous);
 	}
-	
-	@Override
-	final protected void initialize() {
-		if (previous != null && (previous.isRunning() || !previous.isCanceled())) {
+
+	public final void initialize() {
+		if (previous != null && previous.isScheduled()) {
 			previous.cancel();
 		}
-		onInitialize();
+		onInitialize(); // TODO: what should I do with this?
```

Comment:
dont make code comments in this manner

Reply:
what do you mean by what should I do with this?

Reply:
@Daniel-E-B how should I remind myself or ping others about questions? Commit and then comment on the commit?